region/brinstar/pink/Pink Brinstar Hopper Room.json

```diff
@@ -1504,6 +1504,297 @@
       "wallJumpAvoid": true,
       "note": "Shinesparking horizontally into the wall will trigger the elevator as Samus's echos hit it."
     },
+    {
+      "link": [2, 3],
+      "name": "Side Platform Cross Room Jump",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "obstructions": [[3, 0]],
+              "minTiles": 36.3703,
+              "speedBooster": true,
+              "environment": "water",
+              "requires": [
+                "canTrickyDashJump",
+                "canGravityJump",
+                "canMomentumConservingMorph"
+              ],
+              "note": ["This applies to Botwoon Energy Tank Room"],
+              "detailNote": [
+                "This needs extra run speed of $6.0 or $6.1.",
+                "It is easier to do by timing a pause to hit just after jumping and aiming down,",
```

Comment:
- This feels like it could use canInsaneJump, or canBePatient for being so tedius to miss.
- Finding a cue to pause at and jumping a set time after that is probably the setup to go for
- This pause->buffer morph tech looks like something to try on every fast jump height jump like this, where you morph.